### PR TITLE
Add some interesting modern hobbies

### PR DESCRIPTION
**Pull request by personjerry**
#### Summary
Content "A few more interesting hobbies"

#### Purpose of change

Add a few more hobbies, as requested on Reddit. Especially if #50349 goes through, it makes sense that more "average" people with more average social-oriented skills and less "apocalypse" oriented skills will be found.

#### Describe the solution

It's just additional JSON content for hobbies.

#### Describe alternatives you've considered

I've looked through the existing hobbies and professions and open pull requests, I don't believe there are duplicates, but I cannot say I was comprehensive.

#### Testing

I have not tested this at all. I'm assuming this is trivial enough that the existing tests should cover them.

### CODE REVIEW

**Comment by BrettDong** [comment]
The convention for in-game text is to place two spaces between sentences like this:

```diff
-"description": "You studied as a generalist engineer, and you love to tinker. Those skills sure are coming in handy during the apocalypse.",
+"description": "You studied as a generalist engineer, and you love to tinker.  Those skills sure are coming in handy during the apocalypse.",
```

**Comment by personjerry** [comment]
@BrettDong Addressed, thank you!

**Review by LaVeyanFiend — CHANGES_REQUESTED** [review]

File: data/json/hobbies.json [hunk 1]
```diff
@@ -91,5 +91,114 @@
...
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "mixology",
+    "name": "Mixology",
+    "description": "You dabbled in entertaining guests with your homemade drinks.",
+    "points": 1,
+    "proficiencies": [ "prof_food_prep", "prof_winemaking", "prof_beverage_distilling", "prof_brewing" ],
+    "skills": [ { "level": 1, "name": "speech" } ]
```
[the remark this answers is not in the document]
Comment (personjerry) [reply data/json/hobbies.json:148]: I wish there were actual drink proficiencies 😔

File: data/json/hobbies.json [hunk 2]
```diff
@@ -91,5 +91,114 @@
...
+    ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "accounting",
+    "name": "Accounting",
+    "description": "You were always good with numbers and you filed your taxes on time each year.  When you were 11, you audited your parents, and they grounded you.",
+    "points": 2,
+    "traits": [ "PACKMULE" ],
```
Comment (Maleclypse) [inline data/json/hobbies.json:127]: Why packmule?

File: data/json/hobbies.json [hunk 3]
```diff
@@ -91,5 +91,114 @@
...
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "theater",
+    "name": "Theater",
+    "description": "You were part of your local theater troupe and performed in a few plays.",
+    "points": 1,
+    "traits": [ "LIAR" ],
+    "skills": [ { "level": 1, "name": "speech" } ]
```
Comment (Maleclypse) [inline data/json/hobbies.json:138]:
```suggestion
    "skills": [ { "level": 2, "name": "speech" } ]
```

File: data/json/hobbies.json [hunk 4]
```diff
@@ -91,5 +91,114 @@
...
+    "traits": [ "BADCARDIO", "SMELLY" ],
+    "skills": [ { "level": 1, "name": "computer" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "watchmaking",
+    "name": "Watchmaking",
+    "description": "In your free time, you enjoyed tinkering with watches and pocket watches.",
+    "points": 2,
```
Comment (Maleclypse) [inline data/json/hobbies.json:166]:
```suggestion
    "points": 3,
```

File: data/json/hobbies.json [hunk 5]
```diff
@@ -91,5 +91,114 @@
...
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "social_media",
+    "name": "Social Media",
+    "description": "You were popular on social media, while there were still people around to appreciate your content.",
+    "points": 1,
+    "traits": [ "PRETTY" ],
+    "skills": [ { "level": 1, "name": "speech" } ]
```
Comment (Maleclypse) [inline data/json/hobbies.json:178]: Add computers 1 as well.  Social media is computers 1 technically.

**Review by Maleclypse — COMMENTED** [review]
This is good stuff!

File: data/json/hobbies.json (hunk 2 above)
Comment (personjerry) [reply data/json/hobbies.json:127]: You're optimizing the space. It's a form of math problem.

File: data/json/hobbies.json (hunk 3 above)
Comment (personjerry) [reply data/json/hobbies.json:138]: Doesn't this become strictly superior to investing a point in LIAR then?

File: data/json/hobbies.json (hunk 4 above)
Comment (personjerry) [reply data/json/hobbies.json:166]: I thought the initial point invested in a skill becomes 2, so here essentially you'd be paying 1 point for the two proficiencies?

File: data/json/hobbies.json (hunk 3 above)
Comment (Maleclypse) [reply data/json/hobbies.json:138]: I think I got confused and thought it was a 2 point hobby.  Also this is already strictly superior to a point in liar.   But yeah don't move this to two speech.

**Comment by Maleclypse** [comment]
Do you need help in resolving conflicts?  I'm fine with your reasons for what you've done above.

**Comment by personjerry** [comment]
@Maleclypse Oops, I didn't see your response. I kept watchmaking at 2 point cost, kept socialite values, kept packmule, but added the 1 computer skill to social media. I've rolled the changes into one commit and rebased onto master after resolving the conflicts, should be good to go once the tests finish.

**Comment by SariusSkelrets** [comment]
I think the ‘’television’’ hobby would be better as ‘’couch potato’’ or something like that. Television being the hobby’s name sounds weird

The definition would also benefit from adding ‘’,neglecting both your fitness and hygiene’’ after the current end. Everyone can sit and watch some TV once in a while without consequences, but the traits imply unhealthy habits (like doing nothing but watching TV in all their free time)

**Comment by personjerry** [comment]
@SariusSkelrets But "couch potato" isn't a hobby. Watching television is a hobby. Playing video games is a hobby. 🤔 Shouldn't these actually fit in the sentence "My hobby is ______"?

**Comment by SariusSkelrets** [comment]
Being, for example, a heroin addict isn’t what you would call a hobby even if it’s in the hobby category 

For fitting in the phrase: ‘’my hobby is heroin addiction’’. It doesn’t fit at all so it’s not an absolute rule

Television alone won’t ruin your fitness and make you permanently stinky. For that you need really bad lifestyle habits worthy of another name than just television

**Comment by ZhilkinSerg** [comment]
> 
> 
> @SariusSkelrets But "couch potato" isn't a hobby.

And why do you think couch industry is thriving then?

**Pull request closed without merging**